Compatibility with last Vite

Is it possible to make this project compatible with vite@2.6.3 (last version as of today), can't test it on my project. Thanks!

```sh
npm ERR! While resolving: storybook-builder-vite@0.1.0
npm ERR! Found: vite@2.6.3
npm ERR! node_modules/vite
npm ERR!   dev vite@"2.6.3" from the root project
npm ERR!   peer vite@"^2" from @nabla/vite-plugin-eslint@1.3.2
npm ERR!   node_modules/@nabla/vite-plugin-eslint
npm ERR!     dev @nabla/vite-plugin-eslint@"^1.3.2" from the root project
npm ERR!   2 more (vite-plugin-mdx, vite-react-jsx)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"2.5.10" from storybook-builder-vite@0.1.0
npm ERR! node_modules/storybook-builder-vite
npm ERR!   dev storybook-builder-vite@"^0.1.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: vite@2.5.10
npm ERR! node_modules/vite
npm ERR!   peer vite@"2.5.10" from storybook-builder-vite@0.1.0
npm ERR!   node_modules/storybook-builder-vite
npm ERR!     dev storybook-builder-vite@"^0.1.0" from the root project
```

--- a/packages/storybook-builder-vite/package.json
+++ b/packages/storybook-builder-vite/package.json
@@ -25,6 +25,6 @@
         "vue-docgen-api": "^4.40.0"
     },
     "peerDependencies": {
-        "vite": "2.5.10"
+        "vite": ">=2.5.0"
     }
 }

